docker/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN set -eux; \
     case "${arch##*-}" in \
         x86_64 | amd64) ARCH='amd64' ;; \
         ppc64el | ppc64le) ARCH='ppc64le' ;; \
-        aarch64) ARCH='arm64' ;; \
+        aarch64 | arm64) ARCH='arm64' ;; \
```

Comment:
Question -- In what case(s) is arch set to `arm64` instead of `aarch64`?

Reply:
In some machines the `uname -m` does return arm64. This is the case for the Macbooks based on the M1/M2 chips

Reply:
Got it!